presto-server-rpm/src/test/java/io/prestosql/server/rpm/ServerIT.java

```diff
@@ -84,7 +85,12 @@ private static void testServer(String baseImage, String rpmHostPath, String expe
                     .waitingFor(forLogMessage(".*SERVER STARTED.*", 1).withStartupTimeout(Duration.ofMinutes(5)))
                     .start();
 
-            assertServer(container.getContainerIpAddress(), container.getMappedPort(8080), expectedJavaVersion);
+            assertEventually(
```

Comment:
let's split the assertion --
port 8080 should be avail. immediately, not eventually

(i think `SHOW CATALOGS` should respond immediately too)

Reply:
> let's split the assertion --

It used to be splitted. :)